spec/passaporte_web/identity_service_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 describe PassaporteWeb::IdentityService do
   let(:identity) { PassaporteWeb::Identity.find('5e32f927-c4ab-404e-a91c-b2abc05afb56') }
-  let(:mock_identity) { mock('Identity', uuid: 'identity-uuid') }
+  let(:mock_identity) { double('Identity', uuid: 'identity-uuid') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.